components/infra-proxy-service/migrations/migrations.go

```diff
@@ -331,3 +313,12 @@ func (s *MigrationServer) ConfirmPreview(ctx context.Context, req *request.Confi
 		MigrationId: req.MigrationId,
 	}, nil
 }
+
+func StreamErr(err error, ctx context.Context, stream service.MigrationDataService_UploadFileServer, migServer *MigrationServer, migrationId, serverId, errMsg string) {
+	log.Errorf(errMsg)
+	res := handleErrorForUploadFileAndMigration(err, migrationId, serverId, migServer, ctx)
+	errStream := stream.SendAndClose(res)
+	if errStream != nil {
+		log.Errorf(errMsg)
```

Comment:
here you have log the errStream.Error(), with the lo unable to close the stream with this error.

Reply:
I need some more info on this.